<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>633</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFirstRow"/>
   <addaction name="actionLastRow"/>
   <addaction name="actionPrevRow"/>
   <addaction name="actionNextRow"/>
   <addaction name="separator"/>
   <addaction name="actionNewItem"/>
   <addaction name="actionChangeItem"/>
   <addaction name="actionDeleteItem"/>
   <addaction name="actionSubmit"/>
   <addaction name="actionChangePwd"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewItem">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加</string>
   </property>
  </action>
  <action name="actionChangeItem">
   <property name="text">
    <string>修改</string>
   </property>
   <property name="toolTip">
    <string>修改</string>
   </property>
  </action>
  <action name="actionChangePwd">
   <property name="text">
    <string>修改密码</string>
   </property>
   <property name="toolTip">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出登陆</string>
   </property>
  </action>
  <action name="actionFirstRow">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Img/first.png</normaloff>:/new/icon/Img/first.png</iconset>
   </property>
   <property name="text">
    <string>首行</string>
   </property>
  </action>
  <action name="actionLastRow">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Img/last.png</normaloff>:/new/icon/Img/last.png</iconset>
   </property>
   <property name="text">
    <string>尾行</string>
   </property>
  </action>
  <action name="actionPrevRow">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Img/up.png</normaloff>:/new/icon/Img/up.png</iconset>
   </property>
   <property name="text">
    <string>上一条</string>
   </property>
  </action>
  <action name="actionNextRow">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Img/down.png</normaloff>:/new/icon/Img/down.png</iconset>
   </property>
   <property name="text">
    <string>下一条</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Img/remove.png</normaloff>:/new/icon/Img/remove.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionSubmit">
   <property name="text">
    <string>提交</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
